manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -208,4 +208,29 @@ spec:
         {{- range .Values.global.imagePullSecrets }}
         - name: {{ . }}
         {{- end }}
-      {{- end }}
\ No newline at end of file
+      {{- end }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    {{ toJsonMap (omit .Annotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account") | nindent 4 }}
+  labels:
+    {{ toJsonMap .Labels | nindent 4}}
+  name: {{.DeploymentName | quote}}
+  namespace: {{.Namespace | quote}}
+  ownerReferences:
+  - apiVersion: gateway.networking.k8s.io/v1beta1
+    kind: Gateway
+    name: "{{.Name}}"
+    uid: "{{.UID}}"
+spec:
+  clusterIP: None
```

Comment:
I think we should give it a cluster VIP in this implementation and refer to it by VIP.

Reply:
+1, not sure I understand why not have a VIP. 

The waypoint can be also reached explicitly, by an app speaking hbone but not ztunnel PTR-DS.

Reply:
John, can you resolve this comment before merging?